lib/v2/comicat/search.js

```diff
@@ -0,0 +1,40 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const baseUrl = 'https://comicat.org';
+
+module.exports = async (ctx) => {
+    const keyword = ctx.params.keyword;
+    const { data: response } = await got(`${baseUrl}//search.php?keyword=${encodeURIComponent(keyword)}}`);
+    const $ = cheerio.load(response);
+    const list = $('#listTable tbody > tr')
+        .toArray()
+        .map((item) => ({
+                title: $(item).find('td:nth-child(3)').text().trim(),
+                link: `${baseUrl}/${$(item).find('td:nth-child(3) a').attr('href')}`,
+                pubDate: parseDate($(item).find('td:nth-child(1)').text().trim()),
+                category: $(item).find('td:nth-child(2)').text().trim(),
+                author: $(item).find('td:nth-child(8)').text().trim(),
+            }));
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+
+                item.description = `magnet:?xt=urn:btih:${$('#text_hash_id').text().split('，特征码：')[1]}`.trim();
```

Comment:
Adding a magnet link in `description` won't make the RSS feed supported by BT client.
docs: https://docs.rsshub.app/joinus/advanced-feed.html

Reply:
thks, I put it in `enclosure_url` field, and add the page main content as `description`